app/app.go

```diff
@@ -218,7 +218,7 @@ type App struct {
 	DistrKeeper      distrkeeper.Keeper
 	GovKeeper        govkeeper.Keeper
 	CrisisKeeper     crisiskeeper.Keeper
-	UpgradeKeeper    upgradekeeper.Keeper
+	UpgradeKeeper    sdkupgradekeeper.Keeper
```

Comment:
I should also note that we are still selectively using components of the sdk's upgrade module, hence the change in naming to hopefully help differentiate the two. For example, see line 152, where we are using the appupgrade module instead of the sdk's